clustering/singleton/extension/src/main/resources/schema/wildfly-singleton_1_0.xsd

```diff
@@ -56,7 +56,7 @@
                 <xs:documentation>Identifies the cache-container used to back the singleton deployment policy.</xs:documentation>
             </xs:annotation>
         </xs:attribute>
-        <xs:attribute name="cache" type="xs:string" default="default">
+        <xs:attribute name="cache" type="xs:string">
```

Comment:
Can we update the description to indicate that the default-cache of the specified cache container will be used if undefined?